src/routes/index.svelte

```diff
@@ -68,34 +74,43 @@
   {#if brands}
     {#each brands as item}
 			{#if item.post}
-				<Card>
-					<PrimaryAction on:click={() => doAction('openItemPage')}>
-
+				<div style="position: relative;">
+					<Card>
 						<Media
 							style="background-image: url({getImageURL(item)});"
 							aspectRatio="16x9" />
-						<Content>
-							{#if item.phones}
-								<div class="phone-grid">
-									{#each item.phones as phone}
-										<Phone phoneNumber={phone} />
-									{/each}
-								</div>
-							{/if}
-							<p class="caption" on:click={() => item.isCaptionOpen = !item.isCaptionOpen}>
-								{#if item.isCaptionOpen}
-									{item.post.caption}
+							<div class="location-container">
+								{#if item.location}
+									<Location title={item.location.name} address={item.location.address.street} />
```

Comment:
queremos mostrar `zipCode`?

Reply:
what for?

Reply:
no reason; lo pensé porque tenemos el dato, pero aja, igual no es necesario, si no lo queremos, entonces lo podemos quitar del Query

Reply:
No lo quites, quizás sea importante para definir las regiones, sólo no se me hace importante desplegarlo.